src/views/Checkout.vue

```diff
@@ -13,107 +13,16 @@
 </template>
 
 <script lang="ts">
-import {Component, Emit, Prop, Watch, Vue} from 'vue-property-decorator';
-import {AccountSelector, LoginSelector, PaymentInfoLine, SmallPage} from '@nimiq/vue-components';
-import {
-    RequestType,
-    ParsedCheckoutRequest,
-    ParsedSignTransactionRequest,
-    SignTransactionResult,
-} from '../lib/RequestTypes';
-import {AddressInfo} from '../lib/AddressInfo';
-import {KeyInfo, KeyStorageType} from '../lib/KeyInfo';
-import {State, Mutation, Getter} from 'vuex-class';
-import RpcApi from '../lib/RpcApi';
-import {
-    SignTransactionRequest as KSignTransactionRequest,
-    SignTransactionResult as KSignTransactionResult,
-} from '@nimiq/keyguard-client';
-import {ResponseStatus, State as RpcState} from '@nimiq/rpc';
+import {Component, Emit, Vue} from 'vue-property-decorator';
+import {PaymentInfoLine, SmallPage} from '@nimiq/vue-components';
+import {ParsedCheckoutRequest} from '../lib/RequestTypes';
+import {State as RpcState} from '@nimiq/rpc';
 import {Static} from '../lib/StaticStore';
 
 @Component({components: {PaymentInfoLine, SmallPage}})
 export default class Checkout extends Vue {
-    @State private keyguardResult!: KSignTransactionResult | Error | null;
-
     @Static private rpcState!: RpcState;
-    @Static private request!: ParsedSignTransactionRequest;
-    @Static private keyguardRequest!: any;
-    // Uint8Arrays cannot be stored in SessionStorage, thus the stored request has arrays instead and is
-    // thus not the type KSignTransactionRequest
-
-    @Watch('keyguardResult', {immediate: true})
-    private async onKeyguardResult() {
-        if (this.keyguardResult instanceof Error) {
-            //
-        } else if (this.keyguardResult) {
-            // Load web assembly encryption library into browser (if supported)
-            await Nimiq.WasmHelper.doImportBrowser();
-            // Configure to use test net for now
-            Nimiq.GenesisConfig.test();
-
-            let tx: Nimiq.Transaction;
```

Comment:
The whole transaction-creation-code has been moved to the central `TransactionSender` component.